oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -671,11 +671,14 @@ private void addClaimsToOrchSession(OrchSessionItem orchSession, UserInfo userIn
                         AuthUserInfoClaims.VERIFIED_MFA_METHOD_TYPE.getValue(), String.class);
         String rpPairwiseId =
                 userInfo.getClaim(AuthUserInfoClaims.RP_PAIRWISE_ID.getValue(), String.class);
+        String internalCommonSubjectIdentifier = userInfo.getSubject().getValue();
 
         OrchSessionItem updatedOrchSession =
                 orchSession
                         .withVerifiedMfaMethodType(verifiedMfaMethodType)
-                        .withRpPairwiseId(rpPairwiseId);
+                        .withRpPairwiseId(rpPairwiseId)
+                        .withInternalCommonSubjectIdentifier(internalCommonSubjectIdentifier);
+
```

Comment:
Might be worth adding some temp logging like @GHSwallow did below, and using the same process he does for monitoring

Reply:
Sounds good, will do

Reply:
Done